components/query/Validator.ts

```diff
@@ -0,0 +1,24 @@
+import { gql } from "apollo-boost";
```

Comment:
@MonikaCat `apollo-boost` should not be used

```suggestion
import { gql } from "@apollo/client";
```